### PR TITLE
Fix #853 Make explicit the possibility to load a graph by ID or from …

### LINKED ISSUE #853: Make explicit the possibility to load a graph by ID or from a file
A or label should be added between the two labels.

### PR DESCRIPTION
**Pull request by nmco**
…a file

### CODE REVIEW

File: mapcomposer/app/static/translations/it.js [hunk 1]
```diff
@@ -983,6 +983,9 @@ GeoExt.Lang.add("it", {
         closeText: 'Chiudi',
         editChartText: 'Modifica Grafico',
         loadText: "Carica",
-        browseText: "Sfoglia..."
+        browseText: "Sfoglia...",
+        orText: "oppure",
```
Comment (offtherailz) [inline mapcomposer/app/static/translations/it.js]: this is not used anymore

File: mapcomposer/app/static/externals/gxp/src/script/plugins/ChartReporting.js [hunk 2]
```diff
@@ -61,6 +61,9 @@ gxp.plugins.ChartReporting = Ext.extend(gxp.plugins.Tool, {
     loadText: "Load",
     browseText: "Browse...",
     invalidChartText: "Invalid chart",
+    orText: "or",
```
Comment (offtherailz) [inline mapcomposer/app/static/externals/gxp/src/script/plugins/ChartReporting.js]: this is not used anymore

**Comment by offtherailz** [comment]
please attach a screenshot and make mergiable automatically again

**Comment by nmco** [comment]
This issue is fixed by this pull request: https://github.com/geosolutions-it/MapStore/pull/863

**Pull request closed without merging**